### PR TITLE
Update Ruby documentation (2.7.1)

### PR DESCRIPTION
**Pull request by davidcgl**
Adding documentation for Ruby 2.7.

- [x] Updated the versions and releases in the scraper file
- [x] Ensured the license is up-to-date and that the documentation's entry in the array in `about_tmpl.coffee` matches it's data in `self.attribution`
- [x] Ensured the icons and the `SOURCE` file in <code>public/icons/*your_scraper_name*/</code> are up-to-date if the documentation has a custom icon
- [x] Ensured `self.links` contains up-to-date urls if `self.links` is defined
- [x] Tested the changes locally to ensure:
  - The scraper still works without errors
  - The scraped documentation still looks consistent with the rest of DevDocs
  - The categorization of entries is still good

Additional steps:
- [x] Downloaded Ruby 2.7.1: https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.1.tar.bz2
- [x] Compiled Ruby: `./configure --with-openssl-dir="/usr/local/opt/openssl" && make html`
- [x] Copied `ruby-2.7.1/.ext/html` to `devdocs/docs/ruby~2.7`
- [x] `thor docs:generate ruby --force`
- [x] Started the local server: `bundle exec rackup`
- [x] Checked that Ruby 2.7 docs are rendered (see screenshot)

Screen Shot 2020-05-23 at 12 38 28 PM

### CODE REVIEW

**Comment by zicjin** [comment]
@davidcgl May I ask this PR support from https://ruby-doc.org/downloads/ruby_2_7_1_core_rdocs.tgz Download the offline documentation? I download the offline document and execute "thor docs:generate ruby", Then I got the ERROR: Failed to open file: /devdocs/docs/ruby~2.7/table_of_contents.html

**Comment by kristallizer** [comment]
Hey @davidcgl, do you know what's the status of this pull request?

**Comment by jmgarnier** [comment]
Would be nice to have this PR merged ;)

**Comment by davidcgl** [comment]
Based on recent conversation on Gitter [0], there are currently no active maintainers on this project. They are looking for new contributors to take over. Until then, I'm not sure if anyone will approve this PR.

[0] https://gitter.im/FreeCodeCamp/DevDocs

**Review by simon04 — APPROVED** [review]

**Pull request merged**
